elytron/src/test/resources/org/wildfly/extension/elytron/credential-security-factories.xml

```diff
@@ -16,6 +16,7 @@
                                    server="false"
                                    obtain-kerberos-ticket="true"
                                    debug="true"
+                                   mechanisms="KRB5 KRB5LEGACY"
```

Comment:
**It make sense to configure both**: for example following for KRB5+SPNEGO:
`mechanisms="KRB5" mechanism-oids="1.3.6.1.5.5.2"`